test/common/models/Mailbox.js

```diff
@@ -144,8 +150,12 @@ describe('The Mailbox class', function() {
       var mailbox = jmap.Mailbox.fromJSONObject({}, {
         id: 'id',
         name: 'name',
+        namespace: {
+          type: 'personal'
```

Comment:
James is using (currently) `Personal` (Pascal Case) as default value.

Reply:
Done